app/views/orders/new.html.erb

```diff
@@ -3,6 +3,48 @@
   <%= form_for(@order) do |f| %>
```

Comment:
I think simple_form is a pseudo-standard for making forms in rails. What it does is change this:

````erb
<div class="form-group">
  <%= f.label :shipping_address2 %>
  <%= f.text_field :shipping_address2 %>
</div>
````

To this:

````erb
<%= f.field :shipping_address2 %>
````

It automatically makes a div containing a label and a form field. It also infers the form field type (date, string, integer, etc) from the model and puts the right type into the form for you.

It's up to you if you want to use it, but I think it makes things a lot cleaner and easier.